packages/agent/src/agent/http/index.ts

```diff
@@ -473,7 +473,7 @@ export class HttpAgent implements Agent {
     await this.#rootKeyGuard();
     // TODO - restore this value
     const callSync = options.callSync ?? true;
-    const id = await(identity !== undefined ? await identity : await this.#identity);
+    const id = await (identity !== undefined ? await identity : await this.#identity);
```

Comment:
Just came into this while reading through the code.

Can we change this to the following?
```suggestion
    const id = await (identity !== undefined ? identity : this.#identity);
```

Same goes for the createReadStateRequest [below](https://github.com/dfinity/agent-js/pull/1000/files#diff-1deee5c6f68eb82e2be634c32d050e818f857292ed0fa9c9e19026e2ef60869cL1043).

Reply:
I randomly noticed the comment and just wanted to share that if you're already there, you might even be able to take it a step further.

```
const id = await (identity ?? this.#identity);
```

Reply:
Incorporated this into the PR